src/errors/message.js

```diff
@@ -11,10 +11,11 @@ export const MESSAGE = {
     portNumberIsNotInteger:             'A port number should be a valid integer.',
     portsOptionRequiresTwoNumbers:      'The "--ports" option requires two numbers to be specified.',
     portIsNotFree:                      'The specified {portNum} port is already in use by another program.',
-    invalidHostname:                    'The specified "{hostname}" hostname cannot be resolved to the current machine.'
+    invalidHostname:                    'The specified "{hostname}" hostname cannot be resolved to the current machine.',
+    cantFindSpecifiedTestSource:        'Cannot find specified test source file at "{path}".',
```

Comment:
just thought that we don't specify a test source file - we specify the path
that's why we can just say
```
    cantFindSpecifiedTestSource:        'Cannot find a test source file at "{path}".',
```
can't we?